README.md

```diff
@@ -36,8 +36,14 @@ The docs will be under `docs/_build/html`
 
 # To generate an HTML dump of the curriculum
 
-Run `adam.curriculum_to_html parameters/html/curriculum_to_html.phase1.params`
+In English: run `adam.curriculum_to_html parameters/html/curriculum_to_html.phase1.params`
 
+In Chinese: run `adam.curriculum_to_html parameters/html/curriculum_to_html.phase1.chinese.params`
+
+
+# To Run the Learner over the Entire Curriculum
```

Comment:
Perhaps this should say "an Entire Curriculum" rather than the given there are many curricula the learner can run over.